### PR TITLE
[Components Doc] use of new components tels FluentLayout, FluentHeader..

### PR DESCRIPTION
**Pull request by tossnet**
# Pull Request

## 📖 Description

The demo site didn't use the new components, so I replaced the **header** elements with **FluentHeader**, **FluentBody** etc. I also added some CSS styling so that the menu and content had  verticals scrollbars to improve the UX.

### General

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fluentui-blazor/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have added [Unit Tests](https://github.com/Microsoft/fluentui-blazor/blob/master/unit-tests.md) for my new compontent
- [ ] I have modified an existing component
- [ ] I have validate [Unit Tests](https://github.com/Microsoft/fluentui-blazor/blob/master/unit-tests.md) for an existing component

### CODE REVIEW

File: examples/Demo/Shared/Shared/DemoNavMenu.razor.css [hunk 1]
```diff
@@ -1,3 +1,75 @@
-﻿::deep fluent-tree-item {
-    margin: 5px 0;
+﻿/*::deep .fluent-nav-menu {
+    height: Calc(100dvh - 100px);
+    overflow-y: overlay;
+    overflow-x: hidden;
+}*/
+
```
Comment (vnbaaij) [inline examples/Demo/Shared/Shared/DemoNavMenu.razor.css:6]: Please remove commented out code

File: examples/Demo/Shared/Shared/DemoNavMenu.razor.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace FluentUI.Demo.Shared.Shared;
+
+public partial class DemoNavMenu
+{
+    private bool Expanded = true;
```
Comment (vnbaaij) [inline examples/Demo/Shared/Shared/DemoNavMenu.razor.cs:7]: Is this needed?

File: examples/Demo/Shared/Shared/DemoNavMenu.razor.css [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+/* 
+  ##Device = Most of the Smartphones Mobiles (Portrait)
+  ##Screen = B/w 320px to 479px
+*/
+@media (min-width: 320px) and (max-width: 480px) {
+
+    /* CSS */
+
+}
```
Comment (vnbaaij) [inline examples/Demo/Shared/Shared/DemoNavMenu.razor.css:75]: Not sure if we want these here just for a 'if needed' case

**Review by vnbaaij — CHANGES_REQUESTED** [review]

File: examples/Demo/Shared/Shared/DemoNavMenu.razor.cs (hunk 2 above)
Comment (tossnet) [reply examples/Demo/Shared/Shared/DemoNavMenu.razor.cs:7]: no more. I delete this partial class

**Comment by dvoituron** [comment]
I'm having a problem with the latest modifications. The right panel is over the text :-(

image

**Comment by tossnet** [comment]
😱

**Review by dvoituron — APPROVED** [review]

**Pull request merged**